pkgs/tools/misc/mc/default.nix

```diff
@@ -60,7 +60,7 @@ stdenv.mkDerivation rec {
   postFixup = ''
     # remove unwanted build-dependency references
     sed -i -e "s!PKG_CONFIG_PATH=''${PKG_CONFIG_PATH}!PKG_CONFIG_PATH=$(echo "$PKG_CONFIG_PATH" | sed -e 's/./0/g')!" $out/bin/mc
-
+  '' + lib.optionalString (!stdenv.isDarwin) ''
```

Comment:
I don't know if it should be this or `stdenv.isLinux`, maybe somebody else could chime in.

Reply:
Sounds like a good approach. On Linux (and other UNIX-like systems as well, e.g. FreeBSD), we typically have X11 at our disposal as a display manager. On macOS, we only need an X display server in very rare cases.

Reply:
the buildInputs has
```
  ] ++ lib.optionals (!stdenv.isDarwin) [ e2fsprogs gpm ];``` 
	so i followed that